### PR TITLE
Add release notes for columnar processing

### PR DESCRIPTION
**Pull request by nileema**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.130.rst [hunk 1]
```diff
@@ -6,3 +6,7 @@ General Changes
 ---------------
 
 * Add :func:`map_concat` function.
+* Performance improvements for ``FilterAndProject`` operators. This includes projection
+  optimizations for dictionary encoded data. This optimization is turned off by default.
+  It can be configured via ``optimizer.columnar-processing-dictionary`` config property or
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.130.rst]: via the

File: presto-docs/src/main/sphinx/release/release-0.130.rst [hunk 2]
```diff
@@ -6,3 +6,7 @@ General Changes
...
+  ``columnar_processing_dictionary`` session property.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.130.rst]: or the

File: presto-docs/src/main/sphinx/release/release-0.130.rst [hunk 3]
```diff
@@ -6,3 +6,7 @@ General Changes
 ---------------
 
 * Add :func:`map_concat` function.
+* Performance improvements for ``FilterAndProject`` operators. This includes projection
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.130.rst]: FilterAndProject doesn't mean much from a user's point of view. Just say it's an optimization that improves performance for filters and projections, and for dictionary-encoded data.

**Comment by nileema** [comment]
@martint  updated

File: presto-docs/src/main/sphinx/release/release-0.130.rst [hunk 4]
```diff
@@ -6,3 +6,7 @@ General Changes
 ---------------
 
 * Add :func:`map_concat` function.
+* Performance improvements for filter, projections and dictionary encoded data.
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-0.130.rst]: filters

**Comment by martint** [comment]
Looks good

**Comment by nileema** [comment]
merged

**Pull request closed without merging**